code/game/machinery/limbgrower.dm

```diff
@@ -0,0 +1,232 @@
+#define LIMBGROWER_MAIN_MENU       1
+#define LIMBGROWER_CATEGORY_MENU   2
+//use these for the menu system
+
+
+/obj/machinery/limbgrower
+	name = "limb grower"
+	desc = "It grows new limbs using Synthflesh."
+	icon = 'icons/obj/machines/limbgrower.dmi'
+	icon_state = "limbgrower_idleoff"
+	density = 1
+	flags = OPENCONTAINER
+
+	var/operating = 0
+	anchored = 1
+	use_power = 1
+	var/disabled = 0
+	idle_power_usage = 10
+	active_power_usage = 100
+	var/busy = 0
+	var/prod_coeff = 1
+
+	var/datum/design/being_built
+	var/datum/research/files
+	var/list/datum/design/matching_designs
+	var/selected_category
+	var/screen = 1
+
+	var/list/categories = list(
+							"Human",
+							"Lizard",
+							)
+
+	var/mob/living/carbon/human/human
+
+
+
+/obj/machinery/limbgrower/New()
+	..()
+	create_reagents(0)
```

Comment:
I think so?
I'm not quite sure how the reagents system works, so i put that in there (as the same is done on the circuit imprinter)